2025-04-01 15:33:19.754 | INFO     | __main__:main:231 - Using dist with rank 0 (only rank 0 will log)
2025-04-01 15:33:19.755 | INFO     | __main__:main:232 - ****************************************
2025-04-01 15:33:19.755 | INFO     | __main__:main:233 - Starting training with the arguments
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - model_type                     moc
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - run_name                       moc_60m-LR-0.006
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - wandb_project                  moc_pretrain
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - model_config                   configs/llama/llama_60m.json
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - continue_from                  None
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - batch_size                     256
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - gradient_accumulation          2
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - total_batch_size               512
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - max_length                     256
2025-04-01 15:33:19.755 | INFO     | __main__:main:235 - optimizer                      adamw
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - lr                             0.006
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - scheduler                      cosine
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - min_lr_ratio                   0.1
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - activation_checkpointing       False
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - weight_decay                   0.0
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - warmup_steps                   1000
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - eval_every                     1000
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - num_training_steps             10000
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - max_train_tokens               None
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - save_every                     10000
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - save_dir                       checkpoints/llama_60m-2025-04-01-15-33-18
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - tags                           None
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - dtype                          bfloat16
2025-04-01 15:33:19.756 | INFO     | __main__:main:235 - workers                        8
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - seed                           42
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - grad_clipping                  0.0
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - beta1                          0.0
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - single_gpu                     False
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - rank                           128
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - update_proj_gap                50
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - galore_scale                   1.0
2025-04-01 15:33:19.757 | INFO     | __main__:main:235 - proj_type                      std
2025-04-01 15:33:19.757 | INFO     | __main__:main:236 - ****************************************
2025-04-01 15:33:20.392 | INFO     | __main__:main:242 - Shuffling data with seed 42
Traceback (most recent call last):
  File "/home/wt/moc_pretrain/main.py", line 704, in <module>
    main(args)
  File "/home/wt/moc_pretrain/main.py", line 252, in main
    tokenizer = AutoTokenizer.from_pretrained(
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/models/auto/tokenization_auto.py", line 930, in from_pretrained
    config = AutoConfig.from_pretrained(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/models/auto/configuration_auto.py", line 1096, in from_pretrained
    config_dict, unused_kwargs = PretrainedConfig.get_config_dict(pretrained_model_name_or_path, **kwargs)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/configuration_utils.py", line 594, in get_config_dict
    config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/configuration_utils.py", line 653, in _get_config_dict
    resolved_config_file = cached_file(
                           ^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/utils/hub.py", line 266, in cached_file
    file = cached_files(path_or_repo_id=path_or_repo_id, filenames=[filename], **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/utils/hub.py", line 424, in cached_files
    hf_hub_download(
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 862, in hf_hub_download
    return _hf_hub_download_to_cache_dir(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 925, in _hf_hub_download_to_cache_dir
    (url_to_download, etag, commit_hash, expected_size, head_call_error) = _get_metadata_or_catch_error(
                                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 1376, in _get_metadata_or_catch_error
    metadata = get_hf_file_metadata(
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 1296, in get_hf_file_metadata
    r = _request_wrapper(
        ^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 280, in _request_wrapper
    response = _request_wrapper(
               ^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 303, in _request_wrapper
    response = get_session().request(method=method, url=url, **params)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/utils/_http.py", line 96, in send
    return super().send(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
           ^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    response = self._make_request(
               ^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connectionpool.py", line 464, in _make_request
    self._validate_conn(conn)
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connectionpool.py", line 1093, in _validate_conn
    conn.connect()
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connection.py", line 704, in connect
    self.sock = sock = self._new_conn()
                       ^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connection.py", line 198, in _new_conn
    sock = connection.create_connection(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/util/connection.py", line 73, in create_connection
    sock.connect(sa)
KeyboardInterrupt
[rank0]: Traceback (most recent call last):
[rank0]:   File "/home/wt/moc_pretrain/main.py", line 704, in <module>
[rank0]:     main(args)
[rank0]:   File "/home/wt/moc_pretrain/main.py", line 252, in main
[rank0]:     tokenizer = AutoTokenizer.from_pretrained(
[rank0]:                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/models/auto/tokenization_auto.py", line 930, in from_pretrained
[rank0]:     config = AutoConfig.from_pretrained(
[rank0]:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/models/auto/configuration_auto.py", line 1096, in from_pretrained
[rank0]:     config_dict, unused_kwargs = PretrainedConfig.get_config_dict(pretrained_model_name_or_path, **kwargs)
[rank0]:                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/configuration_utils.py", line 594, in get_config_dict
[rank0]:     config_dict, kwargs = cls._get_config_dict(pretrained_model_name_or_path, **kwargs)
[rank0]:                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/configuration_utils.py", line 653, in _get_config_dict
[rank0]:     resolved_config_file = cached_file(
[rank0]:                            ^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/utils/hub.py", line 266, in cached_file
[rank0]:     file = cached_files(path_or_repo_id=path_or_repo_id, filenames=[filename], **kwargs)
[rank0]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/transformers/utils/hub.py", line 424, in cached_files
[rank0]:     hf_hub_download(
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
[rank0]:     return fn(*args, **kwargs)
[rank0]:            ^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 862, in hf_hub_download
[rank0]:     return _hf_hub_download_to_cache_dir(
[rank0]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 925, in _hf_hub_download_to_cache_dir
[rank0]:     (url_to_download, etag, commit_hash, expected_size, head_call_error) = _get_metadata_or_catch_error(
[rank0]:                                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 1376, in _get_metadata_or_catch_error
[rank0]:     metadata = get_hf_file_metadata(
[rank0]:                ^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
[rank0]:     return fn(*args, **kwargs)
[rank0]:            ^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 1296, in get_hf_file_metadata
[rank0]:     r = _request_wrapper(
[rank0]:         ^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 280, in _request_wrapper
[rank0]:     response = _request_wrapper(
[rank0]:                ^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/file_download.py", line 303, in _request_wrapper
[rank0]:     response = get_session().request(method=method, url=url, **params)
[rank0]:                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/requests/sessions.py", line 589, in request
[rank0]:     resp = self.send(prep, **send_kwargs)
[rank0]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/requests/sessions.py", line 703, in send
[rank0]:     r = adapter.send(request, **kwargs)
[rank0]:         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/huggingface_hub/utils/_http.py", line 96, in send
[rank0]:     return super().send(request, *args, **kwargs)
[rank0]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/requests/adapters.py", line 667, in send
[rank0]:     resp = conn.urlopen(
[rank0]:            ^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connectionpool.py", line 787, in urlopen
[rank0]:     response = self._make_request(
[rank0]:                ^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connectionpool.py", line 464, in _make_request
[rank0]:     self._validate_conn(conn)
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connectionpool.py", line 1093, in _validate_conn
[rank0]:     conn.connect()
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connection.py", line 704, in connect
[rank0]:     self.sock = sock = self._new_conn()
[rank0]:                        ^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/connection.py", line 198, in _new_conn
[rank0]:     sock = connection.create_connection(
[rank0]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[rank0]:   File "/home/wt/miniconda3/envs/moc/lib/python3.12/site-packages/urllib3/util/connection.py", line 73, in create_connection
[rank0]:     sock.connect(sa)
[rank0]: KeyboardInterrupt
